src/main/java/de/sakpaas/backend/v2/controller/LocationController.java

```diff
@@ -164,12 +164,12 @@ public LocationController(LocationService locationService,
 
     if (user.isPresent()) {
       occupancyService.save(new Occupancy(location, occupancyReportDto.getOccupancy(),
-          occupancyReportDto.getClientType(), user.get().getId()));
+          occupancyReportDto.getClientType(), null, user.get().getId()));
```

Comment:
wäre es nicht sinnvoller einen zusätzlichen Konstruktor dem Occupanyny hinzuzufügen, anstatt diese unschöne Variante zu nutzen?

Reply:
@benfu99 Ne, ich hätte gerne in der Zukunft, dass wir immer den `RequestHash` mit speichern. Da aber die Methode zum errechnen noch nicht geschrieben ist, musste das erstmal `null` sein :)

Reply:
ah, okay :)